design-toolkit/components/src/components/color-picker/index.tsx

```diff
@@ -125,7 +123,9 @@ function ColorSwatchPickerItem({
       ref={ref}
       id={color.toString('hexa')}
       textValue={color.getColorName(locale)}
-      className='w-fit outline-none outline outline-offset-1 hover:outline-interactive-hover hover:outline-solid focus:outline-interactive-hover focus:outline-solid data-selected:outline-highlight data-selected:outline-solid'
+      className={swatchItem({
+        className: typeof classNames?.item === 'string' ? classNames.item : undefined,
```

Comment:
You can simplify this to just passing `classNames?.item`

Reply:
Done.